prombench/manifests/cluster-infra/3b_prometheus-meta.yaml

```diff
@@ -258,6 +258,7 @@ metadata:
   name: ingress-prometheus-meta
   annotations:
     spec.ingressClassName: "nginx"
```

Comment:
I'm no longer using nginx-ingress controller, but I wasn't aware of this annotation.
I think it was meant to be added under `spec:`

Reply:
Yes, this should be down under `spec:`
```
spec:
  ingressClassName: nginx
```

Reply:
Updated PR.